Cmake Configure:

```
  Completed 'wxwidgets-populate'

C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppBuild.targets(2521,5): warning : Remote deployment might be slow/inefficient. Could not find a part of the path 'D:\Projects\TheAssemblyArmada\Thyme_OmniBl
ade\Win32\Debug\wxwidgets-populate\wxwidgets-populate.recipe'. [D:\Projects\TheAssemblyArmada\Thyme_OmniBlade\.solution\vs2019\_deps\wxwidgets-subbuild\wxwidgets-populate.vcxproj]

  Building Custom Rule D:/Projects/TheAssemblyArmada/Thyme_OmniBlade/.solution/vs2019/_deps/wxwidgets-subbuild/CMakeLists.txt

C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppBuild.targets(2521,5): warning : Remote deployment might be slow/inefficient. Could not find a part of the path 'D:\Projects\TheAssemblyArmada\Thyme_OmniBl
ade\Win32\Debug\ALL_BUILD\ALL_BUILD.recipe'. [D:\Projects\TheAssemblyArmada\Thyme_OmniBlade\.solution\vs2019\_deps\wxwidgets-subbuild\ALL_BUILD.vcxproj]

CMake Error at CMakeLists.txt:274 (add_subdirectory):
  add_subdirectory given source "EXCLUDE_FROM_ALL" which is not an existing
  directory.


Configuring incomplete, errors occurred!
See also "D:/Projects/TheAssemblyArmada/Thyme_OmniBlade/.solution/vs2019/CMakeFiles/CMakeOutput.log".
See also "D:/Projects/TheAssemblyArmada/Thyme_OmniBlade/.solution/vs2019/CMakeFiles/CMakeError.log".
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -268,8 +268,11 @@ if(NOT wxWidgets_FOUND AND BUILD_TOOLS)
         GIT_SHALLOW     TRUE
     )
     
-    FetchContent_GetProperties(wxWidgets)
     FetchContent_MakeAvailable(wxWidgets)
+    if(IS_DIRECTORY "${wxWidgets_SOURCE_DIR}")
+        set_property(DIRECTORY ${wxWidgets_SOURCE_DIR} PROPERTY EXCLUDE_FROM_ALL YES)
+    endif()
+
     set(wxWidgets_FOUND TRUE)
     set(wxWidgets_LIBRARIES wxcore wxbase wxxrc wxxml wxhtml wxadv)
 

